tests/storage/snapshot.go

```diff
@@ -1289,3 +1290,45 @@ func getSnapshotStorageClass(client kubecli.KubevirtClient) (string, error) {
 
 	return "", nil
 }
+
+func AddVolumeAndVerify(virtClient kubecli.KubevirtClient, storageClass string, vm *v1.VirtualMachine, addVMIOnly bool) string {
+	dv := libstorage.NewRandomBlankDataVolume(vm.Namespace, storageClass, "64Mi", corev1.ReadWriteOnce, corev1.PersistentVolumeFilesystem)
+	_, err := virtClient.CdiClient().CdiV1beta1().DataVolumes(dv.Namespace).Create(context.Background(), dv, metav1.CreateOptions{})
+	Expect(err).To(BeNil())
+	Eventually(ThisDV(dv), 240).Should(HaveSucceeded())
```

Comment:
Please use `EventuallyWithOffset` and it might be helpful to do the same for `Expect`